test/Dispatcher.proof.t.sol

```diff
@@ -70,8 +66,11 @@ contract DispatcherIbcWithRealProofs is IbcEventsEmitter, ProofBase {
         Ics23Proof memory proof = load_proof("/test/payload/packet_ack_proof.hex");
 
         // plant a fake packet commitment so the ack checks go through
-        stdstore.target(address(dispatcher)).sig(dispatcher.sendPacketCommitment.selector).with_key(address(mars))
-            .with_key(ch0.channelId).with_key(uint256(1)).checked_write(true);
+        // Stdstore doesn't work for proxies so we have to use store
+        bytes32 slot1 = keccak256(abi.encode(address(mars), uint32(107))); // current nested mapping slot: 107
```

Comment:
how did you calculate the 107?

Reply:
I just used the `forge inspect --storage` command to see what the storage layout is and looked for the mapping from that output. Kind of manual, but probably still easier than setting up the whole handshake process?

Reply:
Manual method is fine. Maybe we can comment on how to get it for future references?

Reply:
added!